tools/scripts/tr-archive/tr35.css

```diff
@@ -29,3 +29,7 @@
 .markdown-alert-important .markdown-alert-title {
     color: blueviolet;
 }
+
+.body h5 {
+    font-variant: small-caps;
```

Comment:
Needs to be the following, otherwise the size is too small (and we want to guarantee it is bold)
```suggestion
    font-variant: small-caps;
    font-size: medium;
    font-weight: bold;
```

Reply:
done, now: 

![image](https://github.com/user-attachments/assets/79f47c8f-f06e-493a-96ff-c929331c8610)